### PR TITLE
adding support for analog outputs for ROS2 Foxy

### PR DESCRIPTION
**Pull request by AlexisFetet**
Adding support for analog outputs on ROS2 Foxy.
Adding a force enable option on analog outputs.

Tested on PhidgetAnalog 1002_0B

### CODE REVIEW

**Comment by mintar** [comment]
Please install pre-commit as described in the README:

https://github.com/ros-drivers/phidgets_drivers#pre-commit-formatting-checks

Then run:

    pre-commit run -a

This will fix the failing build.

File: phidgets_analog_outputs/package.xml
Comment (mintar) [inline phidgets_analog_outputs/package.xml]: If you're adding yourself as a maintainer for the `phidgets_analog_output` package, please watch this repository so that you can actually answer questions by other users, test pull requests, triage bugs etc. about the `phidgets_analog_output` package.

File: phidgets_api/package.xml
Comment (mintar) [inline phidgets_api/package.xml]: Please remove this line.

File: phidgets_msgs/package.xml
Comment (mintar) [inline phidgets_msgs/package.xml]: Please remove this line.

File: phidgets_api/CMakeLists.txt
Comment (mintar) [inline phidgets_api/CMakeLists.txt:23]: Please keep this list sorted alphabetically. The two `analog_output` lines should come before the `digital_input` and after the `analog_inputs` line.

**Review by mintar — CHANGES_REQUESTED** [review]
Thanks for this very useful PR! As a comment to other readers: This is a forward-port of #103 to ROS2.

Additionally, this PR adds a parameter `force_enable`.

I have added some comments inline. Before merging, all commits should be squashed *except* the "force enable" commit, please keep that separate.

**Comment by clalancette** [comment]
@mintar My suggestion here is that we actually retarget this PR to the `rolling` branch, get it reviewed and merged there, and then backport it to Foxy.  That's most consistent with how we do development elsewhere; what do you think?

**Comment by mintar** [comment]
Yes, agreed! This should be retargeted to the `rolling` branch.

**Comment by AlexisFetet** [comment]
I will close this PR and recreate a new one directed to `rolling` with the requested changes to make it cleaner.

**Comment by mintar** [comment]
You don't have to open a new PR! If you continue using this PR, all the info stays in one place. You can force-push to your branch, and it will automatically update here. And you can select the targeted branch (rolling) in the web interface of GitHub.

**Pull request closed without merging**
